tests/h/streamer/websocket_test.py

```diff
@@ -13,6 +14,47 @@
 FakeMessage = namedtuple('FakeMessage', ['data'])
 
 
+class TestMessage(object):
+    def test_reply_adds_reply_to(self, socket):
+        """Adds an appropriate `reply_to` field to the sent message."""
+        message = websocket.Message(socket=socket, payload={'id': 123})
+
+        message.reply({'foo': 'bar'})
+
+        socket.send_json.assert_called_once_with({'ok': True,
+                                                  'reply_to': 123,
+                                                  'foo': 'bar'})
+
+    @pytest.mark.parametrize('ok', [True, False])
+    def test_reply_adds_ok(self, ok, socket):
+        """Adds an appropriate `ok` field to the sent message."""
+        message = websocket.Message(socket=socket, payload={'id': 123})
+
+        message.reply({'foo': 'bar'}, ok=ok)
+
+        socket.send_json.assert_called_once_with({'ok': ok,
+                                                  'reply_to': 123,
+                                                  'foo': 'bar'})
```

Comment:
It might be good to add a test for each of these reply fields that it overrides `ok` and `reply_to` in the payload.